websites/P/Paramount+/presence.ts

```diff
@@ -22,132 +22,157 @@ presence.on("UpdateData", async () => {
+
+				title = data.partOfSeries.name;
+
+				seasonEpisode = `S${data.partOfSeason.seasonNumber}:E${data.episodeNumber} - ${data.name}`;
+
+				if (title) presenceData.details = title;
+
+				if (seasonEpisode) presenceData.state = seasonEpisode;
+
+				presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+				presenceData.smallImageText = video.paused
+					? (await strings).pause
+					: (await strings).play;
+				presenceData.largeImageKey =
+					data.image || "https://i.imgur.com/VWrl85V.jpg";
+
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestampsfromMedia(video);
+
+				if (video.paused) {
+					delete presenceData.startTimestamp;
+					delete presenceData.endTimestamp;
+				}
+			} else if (
+				!vidArea &&
+				document.querySelector('[type="application/ld+json"]') !== null
+			) {
+				presenceData.details = "Viewing series";
+				presenceData.state = JSON.parse(
+					document.querySelector('[type="application/ld+json"]').textContent
+				).name;
+			} else {
+				presenceData.details = "Browsing";
+				presenceData.state = "Viewing Shows";
 			}
+			break;
+		}
 
-			const content = seasonEpi;
-
-			if (movTitle) {
-				presenceData.state = movTitle;
-				presenceData.details = "Watching Movie:";
+		case pathIncludes(path, "/movies"):
+			if (
+				vidArea &&
+				document.querySelector("video.marqueeVideo") &&
+				!vidArea.querySelector("video")
+			) {
+				presenceData.details = "Previewing a movie";
+				presenceData.state = JSON.parse(
+					document.querySelector('[type="application/ld+json"]').textContent
+				).name;
+			} else if (vidArea && vidArea.querySelector("video")) {
+				const movData = JSON.parse(
+					document.querySelector('[type="application/ld+json"]').textContent
+				);
+
+				video = vidArea.querySelector("video");
+
+				if (movData.name) presenceData.state = movData.name;
+				presenceData.details = "Watching a movie";
+
+				presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+				presenceData.smallImageText = video.paused
+					? (await strings).pause
+					: (await strings).play;
+
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestampsfromMedia(video);
+
+				if (video.paused) {
+					delete presenceData.startTimestamp;
+					delete presenceData.endTimestamp;
+				}
+			} else {
+				presenceData.details = "Browsing";
+				presenceData.state = "Viewing Movies";
+			}
+			break;
+
+		case pathIncludes(path, "/live"):
+			liveTitle = document.querySelector(
+				".video__metadata.padded-container > p"
+			).textContent;
+
+			presenceData.details = "Watching Live TV";
+			presenceData.state = liveTitle;
+			presenceData.smallImageKey = Assets.Live;
+
+			break;
+
+		case pathIncludes(path, "/brands"):
+			if (path.includes("/cbs/")) {
+				presenceData.details = "Browsing Brand:";
+				presenceData.state = "CBS";
+				presenceData.largeImageKey = "https://i.imgur.com/BfxpVej.png";
```

Comment:
You have many assets in this presence. Put them in an enum like so: https://github.com/PreMiD/Presences/blob/1f5a4117f92c6549a75e1c01c6305f9c4398b5b5/websites/A/Aodesu/presence.ts#L6-L14